impl/src/main/java/com/okta/sdk/impl/cache/DefaultCache.java

```diff
@@ -126,7 +126,7 @@ public DefaultCache(String name, Map<K, Entry<V>> backingMap) {
      * @see #setTimeToIdle(com.okta.sdk.lang.Duration)
      * @see #setTimeToLive(com.okta.sdk.lang.Duration)
      */
-    public DefaultCache(String name, Map<K, Entry<V>> backingMap, Duration timeToLive, Duration timeToIdle) throws IllegalArgumentException {
+    public DefaultCache(String name, Map<K, Entry<V>> backingMap, Duration timeToLive, Duration timeToIdle) {
```

Comment:
Same comment as above.

Reply:
Same as above `Assert.notNull` does throw a `IllegalArgumentException` it just doesn't belong in the method signature because it is a RuntimeException